src/Chapter1.hs

```diff
@@ -556,8 +560,8 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
-
+isVowel c = elem c  "aeiouy" || elem c "AEIOUY"
```

Comment:
One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎